src/st/dre/e_room_bg.c

```diff
@@ -1,11 +1,38 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "dre.h"
 
-extern ObjInit2 OVL_EXPORT(BackgroundBlockInit)[];
 extern u16 g_EInitCommon[];
 
-// params: Index of ObjInit to use
-//         (== 1) Use a different hardcoded scaleY and scaleX value
+static u8 AnimFrames_80180518[] = {0x40, 0x01, 0xFF, 0x00};
+static u8 AnimFrames_8018051C[] = {
+    0x02, 0x25, 0x02, 0x26, 0x02, 0x27, 0x02, 0x26};
+
+static u8 pad[] = {0, 0, 0, 0};
```

Comment:
I'm not sure if this is the right thing to do, but the data was shifted without it.